test/run.py

```diff
@@ -67,7 +67,7 @@ def cleanup_files(files: typing.Dict):
         ) as proc:
             proc.communicate()
             if proc.returncode != 0:
-                raise Exception("Couldn't create a testfile")
+                print(f"Couldn't remove a testfile {name}")
```

Comment:
Was this hurting in some way? An exception is nice because else the test would fail as a consequence

Reply:
Yeah. I wrote a test with one of the files being deleted as part of the test. This was clashing with my test

Reply:
But it should really ever error if the creation has failed like for example the testfile path is wrong, permission issues or etc, how does it get involved with file deletion?